boot/freeldr/freeldr/ntldr/winldr.c

```diff
@@ -74,6 +74,25 @@ NtLdrOutputLoadMsg(
     }
 }
 
+PACPISDTHeader
+FindAcpiTable()
+{
+    if (!AcpiPresent)
+        return NULL;
+
+    PRSDP_DESCRIPTOR Rsdp = FindAcpiBios();
+    if (!Rsdp)
+        return NULL;
+
+    BOOLEAN is64bitsAcpi = (sizeof(PVOID) > 4) ? (Rsdp->revision > 1) : FALSE;
```

Comment:
`sizeof(PVOID) > 4`:
Could some `#ifdef` be used instead?

`Rsdp->revision > 1`:
`DetectAcpiBios()` uses `Rsdp->revision > 0`. Why the mismatch?

Reply:
Done.

Reply:
i used `Rsdp->revision > 1` to make sure that ACPI version is 2.0 or higher but i think it's not necessary.